### PR TITLE
Preparing release 3.3.1

### PR DESCRIPTION
**Pull request by Natim**
Install `zest.releaser` with the `recommended` dependencies. They contain
`wheel` and `twine`, which are required to release a new version.

    $ pip install "zest.releaser[recommended]"

Step 1
------

     $ git checkout -b prepare-X.Y.Z
     $ prerelease
     $ vim docs/conf.py

- [x] Merge remaining pull requests
- [x] Update ``CHANGELOG.rst``
- [x] If API was updated, update API changelog in :file:`docs/api/index.rst`
- [x] Make sure ``HTTP_API_VERSION`` is up-to-date in kinto/__init__.py``
- [x] Update the link in :file:`docs/configuration/production.rst`
- [x] Update :file:`CONTRIBUTORS.rst`. The following hairy command will output the full list:

```
$ git shortlog -sne | awk '{$1=""; sub(" ", ""); print}' | awk -F'<' '!x[$1]++' | awk -F'<' '!x[$2]++' | sort
```

- [x] Update known good versions of dependencies in ``requirements.txt`` with this command:

```
$ make build-requirements
```
```
$ git commit -a --amend
$ git push origin prepare-X.Y.Z
```

Step 2
------

Once the pull-request is validated, merge it and do a release.
Use the ``release`` command to invoke the ``setup.py``, which builds and uploads to PyPI

    $ git checkout master
    $ git merge --no-ff prepare-X.Y.Z
    $ release
    $ postrelease

Step 3
------

As a final step:

- [ ] Close the milestone in Github
- [ ] Create next milestone in Github in the case of a major release
- [ ] Add entry in Github release page
- [ ] Check that the version in ReadTheDocs is up-to-date
- [ ] Check that a Docker image was built
- [ ] Send mail to ML (If major release)
- [ ] Tweet about it!

Upgrade:

- [ ] Deploy new version on demo server
- [ ] Upgrade dependency in ``kinto-dist`` repo
- [ ] Upgrade version targetted in ``kinto-heroku`` repo
- [ ] Upgrade version of Kinto server for the tests of clients and plugins repos
  (*kinto-http.js, kinto-http.py, kinto-attachment, etc.*)

That's all folks!

### CODE REVIEW

**Comment by n1k0** [comment]
LGTM, I'd tend to say r+ though I'd feel slightly more confident doing so with a go from @leplatrem :)

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-* Adam Chainz <adam@adamj.eu>
 * Aaron Egaas <me@aaronegaas.com>
+* Adam Chainz <adam@adamj.eu>
 * Aditya Bhasin <conlini@gmail.com>
 * Alexis Metaireau <alexis@mozilla.com>
 * Andy McKay <amckay@mozilla.com>
 * Aymeric Faivre <miho@miho-stories.com>
 * Ayush Sharma <ayush.aceit@gmail.com>
 * Balthazar Rouberol <br@imap.cc>
+* Boris FELD <lothiraldan@gmail.com>
```
Comment (leplatrem) [inline CONTRIBUTORS.rst]: nit: Other last names are capitalized but not upercase

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,9 +3,11 @@ Changelog
 
 This document describes changes between each past release.
 
-3.4.0 (unreleased)
+3.3.1 (2016-07-19)
 ==================
 
+**Protocol**
+
 - Add the ``permissions_endpoint`` capability. (#722)
```
Comment (leplatrem) [inline CHANGELOG.rst]: .. when the `kinto.experimental... ` setting is set to true ?

**Comment by leplatrem** [comment]
r+ with nits

**Pull request merged**